### PR TITLE
Default scrape categories and pages

### PR DESCRIPTION
**Pull request by zamattiac**

### CODE REVIEW

File: cli.py [hunk 1]
```diff
@@ -66,8 +66,20 @@ def cli_entry_point(scrape, resume, verify, resume_verify, compile_active, delet
         return
 
     if scrape:
-        if dm is None:
+        if not dm:
```
Comment (adlius) [inline cli.py]: Checking for Boolean value is not the same as checking for None-ness. This is particularly true in Python as None in python is very specific, whereas in language like C++ "Null" is basically equivalent to "0".

File: cli.py [hunk 2]
```diff
@@ -66,8 +66,20 @@ def cli_entry_point(scrape, resume, verify, resume_verify, compile_active, delet
...
+            nodes = True
+            registrations = True
+            users = True
+            institutions = True
+        if nodes and not any([d, f, w, a, r, k]):
+            d = True
+            f = True
+            w = True
+            a = True
+            k = True
```
Comment (adlius) [inline cli.py:82]: Where's "r"?

File: cli.py (hunk 1 above)
Comment (zamattiac) [inline cli.py]: None does evaluate to False, though. Why do we need to insist it be None rather than set to False (if that was even possible in the CLI)?

Comment (adlius) [inline cli.py]: While None evaluates to False, anything other than None and 0 evaluates to True. Specifying it to be None is a safe way of programming. #DefensiveProgramming

**Pull request merged**